bundles/org.smarthomej.binding.notificationsforfiretv/src/main/java/org/smarthomej/binding/notificationsforfiretv/internal/NotificationsForFireTVConfiguration.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2021 Contributors to the SmartHome/J project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.smarthomej.binding.notificationsforfiretv.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link NotificationsForFireTVConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Tom Blum - Initial contribution
+ */
+@NonNullByDefault
+public class NotificationsForFireTVConfiguration {
+
+    public String hostname = "";
+    public int port = 7676;
```

Comment:
If it can't be configured, you could also move it as a constant to `NotificationsForFireTVConnection` (or just include it in the build string in l. 56)

Reply:
done